plugins/woocommerce/templates/emails/customer-pos-completed-order.php

```diff
@@ -31,24 +31,19 @@
  */
 do_action( 'woocommerce_email_header', $email_heading, $email ); ?>
 
-<?php echo $email_improvements_enabled ? '<div class="email-introduction">' : ''; ?>
+<div class="email-introduction">
 <p>
 <?php
 if ( ! empty( $order->get_billing_first_name() ) ) {
 	/* translators: %s: Customer first name */
 	printf( esc_html__( 'Hi %s,', 'woocommerce' ), esc_html( $order->get_billing_first_name() ) );
 } else {
-	printf( esc_html__( 'Hi,', 'woocommerce' ) );
+	printf( esc_html__( 'Hi there,', 'woocommerce' ) );
 }
 ?>
 </p>
-<?php if ( $email_improvements_enabled ) : ?>
-	<p><?php esc_html_e( 'We’ve successfully processed your order, and it’s on its way to you.', 'woocommerce' ); ?></p>
-	<p><?php esc_html_e( 'Here’s a reminder of what you’ve ordered:', 'woocommerce' ); ?></p>
-<?php else : ?>
-	<p><?php esc_html_e( 'We have finished processing your order.', 'woocommerce' ); ?></p>
-<?php endif; ?>
-<?php echo $email_improvements_enabled ? '</div>' : ''; ?>
+<p><?php esc_html_e( 'Here’s a reminder of what you’ve ordered:', 'woocommerce' ); ?></p>
```

Comment:
```suggestion
<p><?php esc_html_e( 'Here’s a reminder of what you’ve bought:', 'woocommerce' ); ?></p>
```

"Ordered" feels strange to me in a POS context, when looking from the shopper's perspective – it sounds as though they have placed an order they expect to be fulfilled in future, when most likely they've got their purchases in hand already.

Reply:
Nice catch, updated in https://github.com/woocommerce/woocommerce/pull/57848/commits/67e2540005cdf1644b23fb1dd0ab8e02135b5a1c:

<img width="529" alt="Screenshot 2025-05-15 at 9 26 30 AM" src="https://github.com/user-attachments/assets/9db3437e-883e-466e-aa14-700b1fda751b" />